Web/Assignment1/YourProfile.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+	<html>
+		<head>
+			<link rel="stylesheet" href="style1.css" type="text/css">
+		</head>
+		<body>
+			<div class="wrapper">
+				<div class="nav">
+					My Profile
+				</div>
+				<div class="nav-left">
+					<img src="C:\Users\Anshita\Downloads\pic.jpg" height="250px"  width="250px">
+				</div>
+				<div class="nav-right">
+					<div class="text-styles">Contact Information</div>
+					</br>
+					E mail: <input type="text" name=" email" value="anshita.arya@metacube.com" maxlength="20" class="print-clean">
+					</br></br>
+					Mobile: <input type="text" name="mobile" value="1111111111" maxlength="10" class="print-clean">
+					</br></br>
+					Work phone: <input type="text" name="work_phone" value="123456" maxlength="10" class="print-clean">
+					</br></br>
+					Internal phone: <input type="text" name="internal_phone" value="12334567" maxlength="10" class="print-clean">
+					</br></br>
+					Skype: <input type="email" name="skype" value="anshita.arya@metacube.com" class="print-clean">
+					</br></br></br>
+					<div class="text-styles">General Information</div>
+					</br>
```

Comment:
ACTION: CHECK "text-styles" css class is present in style1.css. Please check all css used in HTML page.